### PR TITLE
add host as input argument for server

### PR DESCRIPTION
**Pull request by jicuss**
would like to add host input argument to similar to the port argument below
marvin server start --port 8000

### CODE REVIEW

File: src/marvin/cli/server.py [hunk 1]
```diff
@@ -14,6 +14,12 @@ def start(
         "-p",
         help="Port to run the server on",
     ),
+    host: str = typer.Option(
```
Comment (zzstoatzz) [inline src/marvin/cli/server.py:17]: let's add a default here from the marvin settings
```suggestion
    host: str = typer.Option(
        marvin.settings.api_base_url,
```

**Review by zzstoatzz — CHANGES_REQUESTED** [review]
thanks for the PR @jicuss 😄 

just one comment about the default value of the new typer option

**Comment by jicuss** [comment]
reformatting server.py to reflect the standards expected by pre-commit.

**Review by zzstoatzz — APPROVED** [review]
build docs and llm tests are failing because your PR is from a fork, so lgtm!

**Pull request merged**